### PR TITLE
feat: add yarnInstall

### PR DESCRIPTION
**Pull request by u3u**

### CODE REVIEW

File: sao.js [hunk 1]
```diff
@@ -81,6 +81,12 @@ module.exports = {
       message: 'Author name',
       default: ':gitUser:',
       store: true
+    },
+    pm: {
+      message: 'Choose a package manager',
+      choices: ['npm5', 'yarn'],
+      type: 'list',
+      default: 'npm5'
```
Comment (Oliboy50) [inline sao.js]: don't think it's a good idea to put the `npm` version here (will be a PITA to update this after each major release of npm)

**Comment by egoist** [comment]
`yarn install` will use `yarn` if possible while `npmInstall` will use `npm>=5` if possible, how do you think? /cc @Atinux @clarkdo

**Comment by Atinux** [comment]
Can you confirm my changes @egoist ?

https://github.com/nuxt-community/create-nuxt-app/pull/21/commits/db73334e46d13fec649a9b0e2370eede6ae2089a

**Comment by egoist** [comment]
@Atinux LGTM

btw snapshot needs to be updated

**Pull request merged**